### PR TITLE
Added The Play not found feature

### PR DESCRIPTION
**Pull request by hasnainmakada-99**
# Description

Added The play not found feature in the search functionality, Now whenever the user will search for a play which is not there it will throw the play not found messsage

Fixes # [43](https://github.com/atapas/react-play/issues/43)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

`npm start`

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/common/playlists/PlayList.jsx [hunk 1]
```diff
@@ -5,6 +5,13 @@ import './playlist.css';
 
 const PlayList = () => {
   const plays = useSearchFilter();
+  if(Object.keys(plays).length===0){
```
Comment (atapas) [inline src/common/playlists/PlayList.jsx:8]: Isn't the `plays` an array already? Then why to do Object.keys ?

File: src/common/playlists/PlayList.jsx [hunk 2]
```diff
@@ -5,6 +5,13 @@ import './playlist.css';
...
+    return(
+      <>
```
Comment (atapas) [inline src/common/playlists/PlayList.jsx:10]: Wrap it is a div and add a class name.

**Review by atapas — CHANGES_REQUESTED** [review]

File: src/common/playlists/PlayList.jsx (hunk 1 above)
Comment (hasnainmakada-99) [reply src/common/playlists/PlayList.jsx:8]: Yes sir i understand 🤔🤔

**Comment by hasnainmakada-99** [comment]
@atapas sir i've made the changes you told me

File: src/common/playlists/PlayList.jsx [hunk 3]
```diff
@@ -5,6 +5,15 @@ import './playlist.css';
 
 const PlayList = () => {
   const plays = useSearchFilter();
+  if(plays.length===0){
+    return(
+      <>
```
Comment (atapas) [inline src/common/playlists/PlayList.jsx:10]: You do not need the fragments, so remove them.. the top level div that you added will be enough

File: src/common/playlists/PlayList.jsx [hunk 4]
```diff
@@ -5,6 +5,15 @@ import './playlist.css';
 
 const PlayList = () => {
   const plays = useSearchFilter();
+  if(plays.length===0){
```
Comment (atapas) [inline src/common/playlists/PlayList.jsx:8]: Give a space around ===

**Review by atapas — CHANGES_REQUESTED** [review]

**Review by atapas — APPROVED** [review]

**Pull request merged**